src/Features/CSharp/Portable/Completion/KeywordRecommenders/BaseKeywordRecommender.cs

```diff
@@ -49,7 +49,7 @@ private bool IsConstructorInitializerContext(int position, CSharpSyntaxContext c
             if (token.Kind() == SyntaxKind.ColonToken &&
                 token.Parent is ConstructorInitializerSyntax &&
                 token.Parent.IsParentKind(SyntaxKind.ConstructorDeclaration) &&
-                token.Parent.GetParent().IsParentKind(SyntaxKind.ClassDeclaration))
+                token.Parent.Parent.IsParentKind(SyntaxKind.ClassDeclaration))
```

Comment:
i didn't just convert all calls to `GetParent()` with `?.Parent`.  If the preceding checks prove that the left is not null, we can just use `.Parent`.